backend/app/core/security.py

```
@@ -0,0 +1,22 @@
+from fastapi import Depends, HTTPException, status
+from sqlalchemy.orm import Session
+from app.db.session import get_db
+from app.auth.jwt import get_current_user  # your existing function
+from app.schemas.user import Role
+from app.models.user import User
+
+def require_user(db: Session = Depends(get_db), user: User = Depends(get_current_user)) -> User:
```

Ruff (0.12.2)

Unused function argument: `db`

(ARG001)